lib/presentation/widget/myspace/my_space_widget.dart

```diff
@@ -194,7 +194,10 @@ class _MySpaceWidgetState extends State<MySpaceWidget> {
           width: 24,
           height: 24,
           fit: BoxFit.fill,
-      ), onPressed: () => settingModalBottomSheet(context, document)),
+      ), onPressed: () => ContextMenuDocument().build(
```

Comment:
I really want `selectedItem` will be store in viewState. It is for us easier control flow of contextMenu.

Reply:
What do you think of https://github.com/linagora/linshare-mobile-flutter-app/pull/13/commits/b9ea7187569ad4477ba4961c7b418ca04587bba4